### DISCUSSION TITLE
How to make some node_modules packages not use hard links

### DISCUSSION THREAD

**Discussion by AboyL** [pnpm/pnpm#5469] [Q&A]
I have a requirement. After installing a package named Apackage, the code of another package Bpackge will be modified. These two packages are in node_modules. Is there a way to make Bpackge not use hard links? Will not modify the Bpackge code of other projects。

I don't want to use the pnpm patch method, which seems to require rewriting my postinstall script, which feels relatively troublesome

**Comment by zkochan** [comment]
Looks like you need to use patch here.

You can change the way a package is linked to node_modules. You can only change how all packages are linked with the [package-import-method](https://pnpm.io/npmrc#package-import-method) setting.

**Reply by AboyL** [reply]
Will partial package replacements be supported in future releases? package-import-method and patch looks rather troublesome。Because some packages may modify other packages in postinstall。
